i18n/en.json

```diff
@@ -3,6 +3,7 @@
   "info": "Help translate text labels on SVG files. Pick a file on commons to get started.",
   "beta": "This feature is in early testing.",
   "learn-more": "Learn more & leave feedback!",
+  "return-to-home": "← return to home",
```

Comment:
Oooh... an arrow in an i18n message is great fodder for my RTL lectures <3 

Let's not use an arrow -- if we want an arrow, OOUI has icons for it, that also switch directions automatically on direction switch, and it's more proper. I'm worried of trusting translators to remember which direction the arrow should go in anyways.

Reply:
If @prtksxna is already looking at this -- let's change the other places this is used in. OOUI gives us a way to do icons with non framed buttons (so, it's an icon + label that are both clickable) so if we use the arrow/direction icons, it'll resolve the entire problem everywhere and make sure we don't trust translators on knowing where the arrow should actually go?

Reply:
An OOUI widget might work here, but at the moment we're just overriding a single Twig template and so it's not immediately easy to create a widget in PHP. We could do it in JS, but that feels a bit fragile for an error page.

Is it enough to move the arrow out of the message and switch it based on direction?

Reply:
Ah, good point. I guess that would be what OOUI is doing, so it should be okay. Also, there already is an `IconWidget.php` ooui widget, so we could also just use the iconWidget + string in twig itself, if needed; it might be less work?

Either way, let's do the straight-forward easier thing, whichever it is.

Reply:
> we could also just use the iconWidget + string in twig itself

I'm not sure how to create a new object in Twig. I'd imagine we'd need to override whatever's creating and displaying the error template.

Anyway, I've removed the arrow from the message and added the arrow-flipping; see what you think now.